code/firmware/rosco_m68k_v1.2/stage2/main2.c

```diff
@@ -66,13 +66,14 @@ noreturn void lmain() {
 #  endif
 #  ifdef KERMIT_LOADER
     mcPrint("Ready for Kermit receive...\r\n");
-    
+    mcBusywait(100000);
+
     while (!receive_kernel()) {
         mcPrint("\x1b[1;31mSEVERE\x1b[0m: Receive failed; Ready for retry...\r\n");
     }
 
     // Wait a short while for the user's terminal to come back...
-    mcBusywait(100000);
+    mcBusywait(500000);
```

Comment:
Would 400000 do the trick here? That would match the delay that has been added to the current development firmware.